packages/app/test/cypress/integration/20-basic-features/access-to-pagelist.spec.ts

```diff
@@ -55,8 +55,6 @@ context('Access to pagelist', () => {
     cy.getByTestid('page-accessories-modal').parent().should('have.class','show').within(() => {
       cy.get('button.close').eq(1).click();
     });
-
-    cy.screenshot(`${ssPrefix}8-close-page-list-modal`, {capture: 'viewport'});
```

Comment:
what is the reason you deleted this?
it seems tthat the goal for this test is: Successfully expand and `close modal`